### ISSUE TITLE
Missing server coverage

### ISSUE THREAD

**Issue by nycjoker** [lkrnac/primediser#4]
Hi,

I have similar coverage configuration setup and working. I have client and server code instrumented and working fine in the browser. However, I don't get any coverage report on the server side? My client does make ajax requests to node.js for info. I can see in the logs the instrumented server code does get hit. Any idea why I don't get coverage info on the server?

Thanks!

**Comment by lkrnac** [comment]
Yes, I noticed this problem. But when I noticed that I already abandoned this project. Not sure why, sorry.
